.github/pull_request_template.md

```diff
@@ -0,0 +1,56 @@
+Fixes, Closes, or Resolves #
+
+_Please choose one of the keywords above to refer to the issue this PR solves followed by the issue number (e.g. Fixes #000). If no issue number, remove the line. Also, remove everything marked optional that is not applicable. **Remove this note after reading**._
+
+## Changes
+
+- _List the changes_
+
+## Types of changes
+
+#### What types of changes does your code introduce?
+
+- [ ] Bugfix (a non-breaking change that fixes an issue)
+- [ ] New feature (a non-breaking change that adds functionality)
+- [ ] Breaking change (a change that causes existing functionality not to work as expected)
+- [ ] Optimization
+- [ ] Refactoring
+- [ ] Documentation update
+- [ ] Build-related changes
+- [ ] Other: _Description_
+
+## Testing
+
+#### Requires testing
+
+- [ ] Yes
+- [ ] No
+
+#### If yes, did you write tests?
+
+- [ ] Yes
+- [ ] No
+
+#### Notes on testing
+
+_Optional. Remove if not applicable._
+
+## Documentation
+
+#### Requires documentation update
+
+- [ ] Yes
+- [ ] No
+
+_If yes, link the PR to the docs update or the issue with the details labeled `docs`. Remove if not applicable._
+
+#### Requires explanation in Release Notes
+
+- [ ] Yes
+- [ ] No
+
+_If yes, fill in the details here. Remove if not applicable._
```

Comment:
I think it's possible to simplify this
```suggestion
## Testing

- [ ] Tests Required
- [ ] Tests Written (if required)

## Documentation

- [ ] Documentation Update Required
- [ ] Include in Release Notes
```
What do you think?

Reply:
I understand the point for further simplification.

but those questions exist to ensure the contributor is thinking about the content of their contribution in relation to what we typical require from a contribution (e.g tests, docs, updating affected areas etc).

Oversimplification usually doesn't help when writing instructions.

The question and answer format, induces thought, reason being the contributor might not necessarily be internal, seasoned to open source contribution or even familiar with software engineering.